src/View/Console/ExceptionStrategy.php

```diff
@@ -184,7 +184,7 @@ public function prepareExceptionViewModel(MvcEvent $e)
                 if (is_callable($this->message)) {
                     $callback = $this->message;
                     $message = (string) $callback($exception, $this->displayExceptions);
-                } elseif ($this->displayExceptions && $exception instanceof \Exception) {
+                } elseif ($this->displayExceptions && ($exception instanceof \Exception || $exception instanceof \Throwable)) {  // @TODO clean up once PHP 7 requirement is enforced
```

Comment:
Can you open a related issue on [zend-mvc-console](https://github.com/zendframework/zend-mvc-console), please? This functionality is moved into there for the v3 series.